### PR TITLE
DELETE /api/comments/:comment_id

### PR DESCRIPTION
**Pull request by Geno419**

### CODE REVIEW

File: __tests__/test.js [hunk 1]
```diff
@@ -209,3 +206,22 @@ describe("PATCH PATCH /api/articles/:article_id", () => {
...
+  });
+  test("return error 404 when comment_id not in DB", () => {
+    return request(app)
+      .delete("/api/comments/8998")
+      .then((res) => {
+        expect(res.status).toBe(404);
+        expect(res.text).toEqual("8998 is not a comment");
+      });
+  });
+});
```
Comment (alice-y-codes) [inline __tests__/test.js:227]: Missing test Status 404, non existent ID, e.g 999

File: db/controllers/controller.js [hunk 2]
```diff
@@ -92,7 +82,17 @@ exports.patchByArticleId = (req, res, next) => {
...
+};
+
+exports.deleteCommentByID = (req, res, next) => {
+  const { comment_id } = req.params;
+  verifyComment(comment_id, res, next)
+    .then(() => {
+      return removeComment(comment_id, next);
+    })
+    .then((data) => {
+      res.status(200).send(data);
```
Comment (alice-y-codes) [inline db/controllers/controller.js:95]: no need to send data

**Review by alice-y-codes — COMMENTED** [review]
see comments

File: __tests__/test.js (hunk 1 above)
Comment (Geno419) [reply __tests__/test.js:227]: I don't think I understand, isn't that test testing returning error 404 when comment_id not in DB

Comment (nc-poonam-rajput) [reply __tests__/test.js:227]: True this is already covered
You are missing a invalid ID though status 400

**Pull request merged**